code/modules/sanity/sanity_lists.dm

```diff
@@ -81,7 +81,13 @@
 		"But they know, they always know. Accept this.",
 		"Something inside your skull cracks. You feel no pain.",
 		"Curiosity is your greatest sin.",
-		"A burger would be good. Anything edible, really."
+		"A burger would be good. Anything edible, really.",
+		"What can possibly be gained by upsetting the current order?",
+		"You have chosen your answer. Now see it through.",
+		"Noone cared who I was until I put on the scarf.",
+		"Pills. Vape. Inject.",
+		"The snow. It calls to me... but the warm holds me tightly.",
```

Comment:
```suggestion
		"The cold. It calls to me... but the warm holds me tightly.",
```
Sounds more ominous. And we don't have snow on Eris so it's kinda weird to blurt that out of nowhere.

Reply:
Moebius Chemistry disagrees